pkg/lib/infra/sms/twilio/client.go

```diff
@@ -168,7 +168,7 @@ func (t *TwilioClient) makeError(
 				"Detail": errorCode,
 			}), err)
 	case 30002:
-		err = errors.Join(smsapi.ErrKindAuthorizationFailed.NewWithInfo(
+		err = errors.Join(smsapi.ErrKindDeliveryRejected.NewWithInfo(
 			"twilio authorization failed", apierrors.Details{
```

Comment:
The message needs update as well. Please also cross-check other providers.

Reply:
Should be fixed in later commits.